docs/source/guides/contribution_guide.rst

```diff
@@ -113,7 +113,7 @@ Pull Request Conflicts
 ----------------------
 
 As an effort to preserve authorship and a cohesive commit history, we prefer if proposed contributions
-are rebased over master (or appropriate branch) when a merge conflict arises,
+are rebased over `main` (or appropriate branch) when a merge conflict arises,
```

Comment:
```suggestion
are rebased over ``main`` (or appropriate branch) when a merge conflict arises,
```

Reply:
I do not see this diff locally :thinking: